spec/support/controller_macros.rb

```diff
@@ -0,0 +1,50 @@
+module ControllerMacros
```

Comment:
> Refactor testing of devise auth into controller_macros.rb

what does this tell me? 
i do not know controller macros. what is it for and how do we benefit from?

Reply:
Those functions are now available in all `ControllerSpecs`. It gets included in `rails_helper.rb`